test/IISIntegration.FunctionalTests/Utilities/Helpers.cs

```diff
@@ -37,5 +38,15 @@ public static void ModifyAspNetCoreSectionInWebConfig(DeploymentResult deploymen
             element.SetAttributeValue(key, value);
             config.Save(webConfigFile);
         }
+
+        public static async Task<DeploymentResult> DeployApplication(IApplicationDeployer deployer, string ancmVersion = "V2")
+        {
+            var deploymentResult = await deployer.DeployAsync();
+            if (ancmVersion == "V2")
```

Comment:
Why is it a string?

Reply:
Enum? Verify it's V1 or V2

Reply:
Based on your offline feedback, we should put this check in the deployer itself rather than in the tests?

Reply:
Yeah